### DISCUSSION TITLE
Tables groups

### DISCUSSION THREAD

**Discussion by Nightmare53** [dineug/erd-editor#253] [Ideas]
Hi great tool to make an ERD.

I think you may add some tools to make groups of table, to separate the DB into modules for exemple.

I have all my users modules table and in another group i would like to set all my roles/rights tables who are in another module.
By set some colors on table or make some background square for exemple

**Comment by dineug** [comment]
The color function already exists

![Nov-08-2021 20-28-10](https://user-images.githubusercontent.com/45829489/140734407-b874afdd-86bd-4ed0-a231-5318a810ad8c.gif)
<img width="534" alt="스크린샷 2021-11-08 오후 8 27 31" src="https://user-images.githubusercontent.com/45829489/140734446-b0298a47-eb6c-4e39-8400-c6aa3ca5834a.png">
